virttest/utils_test/qemu/__init__.py

```diff
@@ -725,6 +725,10 @@ def check_memory(self, vm=None):
         utils_misc.verify_dmesg()
         self.os_type = self.params.get("os_type")
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
+        # Notes:
+        #    for memory hotplug, some os need to wait several seconds to
+        #  let system perceive it.
+        time.sleep(wait_time)
```

Comment:
Shouldn't it be moved before get `guest_mem_size` ?  Sleep and then get total mem in guest os .

Reply:
`vm_mem_size = self.get_vm_mem(vm)` after sleep is just get mem from automation level, logic parameter.

Reply:
Update,thx.